src/core/css/ErrorHandler.scss

```diff
@@ -7,4 +7,8 @@
   padding: 10px;
   margin: 0;
   margin-bottom: 10px;
+
+  > li {
```

Comment:
Usually we assign class names and just target the elements directly. I think there are some silly performance advantages to that in browsers too, but largely it's just our convention. So this would be `.ErrorHandler-listItem` or whatever.